### DISCUSSION TITLE
Getting errors with WebView2 since updating nuget packages

### DISCUSSION THREAD

**Discussion by Agredo** [dotnet/maui#24983] [Q&A]
Did someone get these errors already?

I updated my NuGet packages. I don’t use WebView in my project and would like to avoid it. But I don’t know which package now targets it…

Maybe someone already had this issue and can help.

Thank you :)

![image](https://github.com/user-attachments/assets/a14c55b7-c9b6-4512-bf4b-e1b3ce45a2b5)

**Comment by Akshay060606** [comment]
Well, I also faced the same issue. I just had to clone my project. I couldn't solve it. I wonder how did you solve that?
